bal_addresses/addresses.py

```diff
@@ -31,14 +31,17 @@ class NoResultError(Exception):
     pass
 
 class AddrBook:
-
     fullbook = requests.get(f"{GITHUB_RAW_OUTPUTS}/addressbook.json").json()
-    chains = DotMap(requests.get(
+    chains = Munch(requests.get(
         "https://raw.githubusercontent.com/BalancerMaxis/bal_addresses/main/extras/chains.json"
     ).json())
+    CHAIN_IDS_BY_NAME = chains["CHAIN_IDS_BY_NAME"]
```

Comment:
If you were saying that CHAIN_IDS_BY_NAME is a constant, it is not. It's actually a dict from dynamically fetched data. I recommend keeping it within the class then, but just make it lowercase :)

Reply:
Good advice, will adopt later, just merging this to get this working/base indexed.